cloud_utils/scheduler/kubernetes_connector.py

```diff
@@ -134,6 +134,9 @@ def create_cron_job(
 def _make_pod_manifest(
```

Comment:
I suggest adding a docstring with a usage example here. Since it's getting a bit complicated.